workflow.rst

```diff
@@ -378,11 +378,34 @@ order:
     * ``workflow.[workflow name].announce``
     * ``workflow.[workflow name].announce.[transition name]``
 
+    You can avoid triggering those events by using the context::
+
+        $workflow->apply($subject, $transitionName, [Workflow::DISABLE_ANNOUNCE_EVENT => true]);
+
+    .. versionadded:: 5.1
+
+        The ``Workflow::DISABLE_ANNOUNCE_EVENT`` constant was introduced in Symfony 5.1.
+
+    .. versionadded:: 5.2
+
+        The context will be accessible in all the events::
```

Comment:
`In Symfony 5.2, the context is accessible in all events::`

Reply:
Done !